boards/riscv32/hifive1/doc/hifive1.rst

```diff
@@ -0,0 +1,60 @@
+.. _hifive1:
+
+SiFive HiFive1
+##############
+
+Overview
+********
+
+The HiFive1 is a development board with the FE310 RISC-V SoC.
```

Comment:
I'd merge these first two sentences:

    The HiFive1 is an Arduino-compatible development board with an FE310 RISC-V SoC.